bot/prisma/migrations/20230406213618_initial_migration/migration.sql

```diff
@@ -0,0 +1,58 @@
+-- CreateTable
+CREATE TABLE "SoundCommand" (
```

Comment:
Huh - are the commands in Airhorn dynamic?

Reply:
They are and always have been since v2 (ref: the current `bot/config[.example].json`). This just moves that data into the database so it can be easily referenced using foreign keys, also makes it much easier to expand the bot to have new features (such as leaderboards, "top usages of x sound").